server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/LocalDelivery.java

```diff
@@ -118,7 +118,6 @@ public void init(MailetConfig mailetConfig) throws MessagingException {
             .userRepository(usersRepository)
             .resourceLocator(ResourceLocatorImpl.instanciate(usersRepository, sieveRepository))
             .consume(getInitParameter("consume", true))
-            .deliveryHeader("Delivered-To")
```

Comment:
do you mean that there's no configuration entry for that ? Would it make sense to have one ?